runtime/fiber.c

```diff
@@ -587,6 +587,16 @@ void caml_free_stack (struct stack_info* stack)
   }
 }
 
+void caml_free_gc_regs_buckets(value *gc_regs_buckets)
+{
+  while (gc_regs_buckets != NULL) {
+    value *next = (value*)gc_regs_buckets[0];
+    caml_stat_free(gc_regs_buckets);
+    gc_regs_buckets = next;
+  }
+}
```

Comment:
Note: @Octachron and myself discussed this offline and we decided that it would be nice to have a documentation comment that explains what `gc_regs` and `gc_regs_buckets` are. I may submit this as a separate (complementary) PR.

Reply:
Yes, some documentation about `gc_regs_buckets` would be great.  I know what `gc_regs` does in OCaml 4, but I'm a bit lost as to the OCaml 5 implementation of `gc_regs`, what with the allocation via malloc and so on.

Reply:
For the record, I proposed some documentation of `gc_regs` and `gc_regs_buckets` in #11437.